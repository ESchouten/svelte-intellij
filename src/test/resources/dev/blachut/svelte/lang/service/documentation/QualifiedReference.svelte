<script lang="ts">
  let local = 323;
</script>

{local.<caret>toFixed()}