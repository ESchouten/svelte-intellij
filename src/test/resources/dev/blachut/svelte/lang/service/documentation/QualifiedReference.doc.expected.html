<div class='definition'><pre><span style="color:#7a7a43;">Number.toFixed</span><span style="">(<br></span><span style="">&#32;&#32;&#32;&#32;fractionDigits?:&#32;</span><span style="color:#000080;">number</span><span style="">):&#32;</span><span style="color:#000080;">string</span></pre></div><div class='content'><p>The <strong><code>toFixed()</code></strong> method formats a number using fixed-point
notation.</p></div>
<table class='sections'>
<tr><td valign='top' class='section'><p>Params:</td><td valign='top'><p><code>digits</code> &ndash; The number of digits to appear after the decimal point; this may be a value between <code>0</code> and <code>20</code>, inclusive, and implementations may optionally support a larger range of values. If this argument is omitted, it is treated as <code>0</code>.<br><span style='font-size:0.2em'>&nbsp;</span>
</td>
<tr><td valign='top' class='section'><p>Returns:</td><td valign='top'><p>A string representing the given number using fixed-point notation.</p></td>
<tr><td valign='top' class='section'><p>Throws:</td><td valign='top'><p><code>RangeError</code> &ndash; If <code>digits</code> is too small or too large. Values between <code>0</code> and <code>100</code>, inclusive, will not cause a <a href="https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/RangeError"><code>RangeError</code></a>. Implementations are allowed to support larger and smaller values as chosen.<br><span style='font-size:0.2em'>&nbsp;</span>
<p><code>TypeError</code> &ndash; If this method is invoked on an object that is not a <a href="https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Number"><code>Number</code></a>.<br><span style='font-size:0.2em'>&nbsp;</span>
</td>
<tr><td valign='top' class='section'><p>Supported by:</td><td valign='top'>Chrome, Chrome Android, Edge, Firefox, IE 5.5, Node.js, Opera 7, Safari 2, Safari iOS</td><tr><td valign='top' colspan='2'><icon src='JavaScriptPsiIcons.FileTypes.TypeScriptFile'/>&nbsp;built-in, es5</td></table>
<div class='content'>By <a href='https://developer.mozilla.org/en-us/docs/web/javascript/reference/global_objects/number/tofixed/contributors.txt'>Mozilla Contributors</a>, <a href='https://creativecommons.org/licenses/by-sa/2.5/'>CC BY-SA 2.5</a></div>
